lib/output/writer/redis_pubsub.go

```diff
@@ -44,7 +46,7 @@ type RedisPubSubConfig struct {
 func NewRedisPubSubConfig() RedisPubSubConfig {
 	return RedisPubSubConfig{
 		URL:     "tcp://localhost:6379",
-		Channel: "benthos_chan",
+		Channel: "benthos_chan_${!metadata:channel}",
```

Comment:
Unfortunately, this needs to stay as `benthos_chan` by default otherwise it's breaking change.

Reply:
reverted this, pushed